### PR TITLE
Proposing a new Wiki article about browser storage

### PR DESCRIPTION
**Pull request by undefinedsymbol**
This article is aimed at explaining how to store in localStorage and sessionStorage.

### CODE REVIEW

**Comment by raisedadead** [comment]
@undefinedsymbol Nice work! Thanks for a wiki article on this topic.

You need to rename the article from `browser-storage` to `Browser-Storage.md` as per our [pull request guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/PULL_REQUEST_TEMPLATE)

I have a few more suggestions to make regarding the grammar and wording as well, once you change the name.

**Comment by raisedadead** [comment]
@undefinedsymbol If you need any help then please ping us in the [wiki chat room](https://gitter.im/FreeCodeCamp/Wiki)

**Comment by undefinedsymbol** [comment]
Hi,

I just changed the name.
Please let me know about any suggestions about grammar/wording.

Thanks,
Daniela

File: Browser-Storage.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+## Store data in your web browser's storage
```
Comment (raisedadead) [inline Browser-Storage.md]: Store data in ~~your~~ web browser's storage

File: Browser-Storage.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+In order to store data coming from and created by your web application, you do not necessarily need a database.
```
Comment (raisedadead) [inline Browser-Storage.md]: `store data coming from and created by` > `manage data handled in`

File: Browser-Storage.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+The respective features are supported by Chrome (version 4 and higher), Mozilla Firefox (version 3.5 and higher) and Internet Explorer (version 8 and higher), and a range of other browsers including those of iOs and Android.
```
Comment (raisedadead) [inline Browser-Storage.md]: `The respective features are` > `Browser Local Storage features are`

Comment (raisedadead) [inline Browser-Storage.md]: `iOs` > `iOS`

File: Browser-Storage.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+There are two main possibilities for browser storage:
+
+### 1. localStorage
+
+Any content/data saved to the localStorage object will be available after the browser has been restarted (closed and opened again).
```
Comment (raisedadead) [inline Browser-Storage.md]: Format the keywords such as "localStorage" as code using back ticks. For ex. `localStorage` 
Do this else where

File: Browser-Storage.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+In order to ***save an item*** to localStorage, you can use the method "setItem". This method must be handed a key and a value.
```
Comment (raisedadead) [inline Browser-Storage.md]: "setItem" > `setItem()`

File: Browser-Storage.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+```
+Example: localStorage.setItem("mykey","myvalue");
+```
+To ***retrieve the item from the localStorage***, the method getItem must be used. The getItem method must be handed the key of the data you would like to retrieve:
```
Comment (raisedadead) [inline Browser-Storage.md]: getItem > `getItem` and elsewhere

File: Browser-Storage.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+  ```
+  Example: localStorage.clear();
+  ```
+
+### 2. sessionStorage
+
+Items saved in the sessionStorage object will remain until the browser is closed by the user. Then, the storage will be cleared.
+
+You can save an item to sessionStorage, please use the method "setItem" on the sessionStorage object:
```
Comment (raisedadead) [inline Browser-Storage.md]: > You can save an item to sessionStorage, please 

to 

> When you save an item to `sessionStorage`, use

"setItem" > `setItem()` and else where

File: Browser-Storage.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+```
+Example: sessionStorage.setItem("mykey","myvalue");
+```
+To ***retrieve the item from the sessionStorage***, the method getItem must be used. The getItem method must be handed the key of the data you would like to retrieve:
+
+  ```
+  Example: sessionStorage.getItem("mykey");
+  ```
+
+  You can remove an item from sessionStorage by using the removeItem method. This method must be handed the key of the item to be removed:
```
Comment (raisedadead) [inline Browser-Storage.md]: Refer earlier comment about rewording

File: Browser-Storage.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+We can now save it to localStorage or sessionStorage using the setItem method:
+```
+localStorage.setItem("ourarraykey",JSON.stringify(ourArray));
+```
+or, for sessionStorage:
+```
+sessionStorage.setItem("ourarraykey",JSON.stringify(ourArray));
+```
+
+In order to be saved, the array must first be converted to a string. In our above shown example, we are using the JSON.stringify method to accomplish this.
```
Comment (raisedadead) [inline Browser-Storage.md]: >  In our above shown example

to

>  In the example shown above,

File: Browser-Storage.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+When retrieving our data from the localStorage or sessionStorage, we will convert it back to an array:
```
Comment (raisedadead) [inline Browser-Storage.md]: ~we will~

**Comment by raisedadead** [comment]
@undefinedsymbol Please check the comments and address them.
Once you have done we need you to [Squash your Commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one).

Since this is your first time, Let us know if you need any help with the squashing by pinging us in the [Chat Room](https://gitter.im/FreeCodeCamp/Wikki)

**Pull request merged**